modules/openapi-generator/src/test/resources/3_0/oneOfDiscriminator.yaml

```diff
@@ -170,6 +170,37 @@ components:
           type: integer
       oneOf:
         - $ref: '#/components/schemas/FruitType'
+    FruitTypeEnum:
+      type: string
+      enum: [APPLE, BANANA]
+    FruitOneOfEnumMappingDisc:
+      type: object
+      properties:
+        fruitType:
+          $ref: "#/components/schemas/FruitTypeEnum"
+      required: fruitType
```

Comment:
The required property must be an array.

Reply:
@cachescrubber why do you say that? In the documentation here: https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/ all the examples have a required discriminator field of type `string`. I actually thought that I'd read somewhere that it must be string, but I can't find that reference now.

Reply:
Sorry, I see the problem now. I've pushed a fix.